src/commands/RequestDiscordAccessCommand.ts

```diff
@@ -0,0 +1,86 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction, GuildMember } from 'discord.js';
+import { ExecuteFunction } from '../interfaces/Command';
+import * as students from '../../assets/Listado.json';
+
+function emailIsValid(email: string) {
+    const re =
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(String(email).toLowerCase());
+}
+
+function padronIsValid(padron: string) {
+    const minimumPadronLength = 5;
+    const maximumPadronLength = 6;
+    return (
+        padron.length >= minimumPadronLength &&
+        padron.length <= maximumPadronLength &&
+        !isNaN(parseInt(padron))
+    );
+}
+
+function getStudentInfo(email: string, padron: string) {
+    for (const index in students) {
+        if (
+            students[index].includes(email) &&
+            students[index].includes(padron)
+        ) {
+            return students[index];
+        }
+    }
+    return [];
+}
```

Comment:
Si no entiendo mal, `students` es una lista donde cada elemento es una lista que contiene numero de estudiante, email, etc..., por lo cual esta funcion tendria como tipo de retorno `any[]`.
Dado que esta funcion solo se utiliza para revisar si existe un estudiante con esos datos, no convendria que esta funcion retornara un booleano si el usuario es valido o no?

```suggestion
function studentIsValid(email: string, padron: string): boolean {
    return students.some(student => {
        return student.includes(email) && student.includes(padron)
    });
}
```

Reply:
Totalmente! La función había quedado de un copy paste del código usado en `RequestGroupRoleCommand.ts`que tiene lógica similar pero además hace uso de los datos de `studentsInfo`. Ahora mismo modifico el código con esta sugerencia